diff -urNad xamarin-gtk-theme-master.old/autogen.sh xamarin-gtk-theme-master/autogen.sh
--- xamarin-gtk-theme-master.old/autogen.sh	2017-05-24 11:51:50.000000000 -0400
+++ xamarin-gtk-theme-master/autogen.sh	2019-02-19 10:34:51.352836848 -0500
@@ -16,7 +16,10 @@
 	DIE=1
 }
 
-if automake-1.15 --version < /dev/null > /dev/null 2>&1 ; then
+if automake-1.16 --version < /dev/null > /dev/null 2>&1 ; then
+	AUTOMAKE=automake-1.16
+	ACLOCAL=aclocal-1.16
+elif automake-1.15 --version < /dev/null > /dev/null 2>&1 ; then
 	AUTOMAKE=automake-1.15
 	ACLOCAL=aclocal-1.15
 elif automake-1.14 --version < /dev/null > /dev/null 2>&1 ; then
